packages/medusa-core-utils/src/get-config-file.js

```diff
@@ -14,6 +14,7 @@ function getConfigFile(rootDir, configName) {
     configFilePath = require.resolve(configPath)
     configModule = require(configFilePath)
   } catch (err) {
+    console.errror(err)
```

Comment:
issue: Typo in `errror`
```suggestion
    console.error(err)
```